h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -6,8 +6,8 @@
 import hex.ModelBuilder;
 import hex.StackedEnsembleModel;
 import hex.deeplearning.DeepLearningModel;
-import hex.deepwater.DeepWater;
-import hex.deepwater.DeepWaterParameters;
+//import hex.deepwater.DeepWater;
```

Comment:
Can we just outright delete it? I don't think it will be ever needed

Reply:
Yes, will delete.